// leaflet/src/layer/index.js
export {Layer} from 'leaflet/src/layer/Layer';
export {LayerGroup, layerGroup} from 'leaflet/src/layer/LayerGroup';
export {FeatureGroup, featureGroup} from 'leaflet/src/layer/FeatureGroup';
import {GeoJSON, geoJSON, geoJson, geometryToLayer, coordsToLatLng, coordsToLatLngs, latLngToCoords, latLngsToCoords, getFeature, asFeature} from 'leaflet/src/layer/GeoJSON';
GeoJSON.geometryToLayer = geometryToLayer;
GeoJSON.coordsToLatLng = coordsToLatLng;
GeoJSON.coordsToLatLngs = coordsToLatLngs;
GeoJSON.latLngToCoords = latLngToCoords;
GeoJSON.latLngsToCoords = latLngsToCoords;
GeoJSON.getFeature = getFeature;
GeoJSON.asFeature = asFeature;
export {GeoJSON, geoJSON, geoJson};

export {ImageOverlay, imageOverlay} from 'leaflet/src/layer/ImageOverlay';
export {VideoOverlay, videoOverlay} from 'leaflet/src/layer/VideoOverlay';
export {SVGOverlay, svgOverlay} from 'leaflet/src/layer/SVGOverlay';

export {DivOverlay} from 'leaflet/src/layer/DivOverlay';
export {Popup, popup} from 'leaflet/src/layer/Popup';
export {Tooltip, tooltip} from 'leaflet/src/layer/Tooltip';

export * from 'leaflet/src/layer/marker';
export * from 'leaflet/src/layer/tile';
export * from 'leaflet/src/layer/vector';
